json-jackson/shared/src/main/scala/rapture/json-jackson/ast.scala

```diff
@@ -22,7 +22,7 @@ import rapture.data.DataTypes
 import com.fasterxml.jackson.databind._
 import com.fasterxml.jackson.databind.node.NullNode
 
-import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
```

Comment:
Yep. In scala 2.12 JavaConversions was deprecated (https://github.com/scala/scala/blob/2.12.x/src/library/scala/collection/JavaConversions.scala), so I've used `JavaConverters` and explicit conversion